arangod/RocksDBEngine/RocksDBEngine.cpp

```diff
@@ -1134,6 +1157,37 @@ ErrorCode RocksDBEngine::getCollectionsAndIndexes(TRI_vocbase_t& vocbase,
   return TRI_ERROR_NO_ERROR;
 }
 
+ErrorCode RocksDBEngine::getReplicatedLogs(TRI_vocbase_t& vocbase,
+                                           arangodb::velocypack::Builder& result) {
+  rocksdb::ReadOptions readOptions;
+  std::unique_ptr<rocksdb::Iterator> iter(
+      _db->NewIterator(readOptions, RocksDBColumnFamilyManager::get(
+          RocksDBColumnFamilyManager::Family::Definitions)));
+
+  result.openArray();
+
+  auto rSlice = rocksDBSlice(RocksDBEntryType::ReplicatedLog);
+
+  for (iter->Seek(rSlice); iter->Valid() && iter->key().starts_with(rSlice); iter->Next()) {
+    if (vocbase.id() != RocksDBKey::databaseId(iter->key())) {
+      continue;
+    }
+
+    auto slice = VPackSlice(reinterpret_cast<uint8_t const*>(iter->value().data()));
+
+    if (arangodb::basics::VelocyPackHelper::getBooleanValue(slice, StaticStrings::DataSourceDeleted,
+                                                            false)) {
+      continue;
+    }
+
+    result.add(slice);
+  }
+
+  result.close();
+
+  return TRI_ERROR_NO_ERROR;
```

Comment:
The method currently always returns `TRI_ERROR_NO_ERROR`. Should we make it void?